tcms/testplans/templates/testplans/get.html

```diff
@@ -168,14 +168,20 @@ <h2 class="card-pf-title">
                                                     {% endfor %}
                                                     </ul>
                                                 </li>
-
-                                                <li>
-                                                    <a id="default-tester-button" href="#">
+                                                <li class="dropdown-submenu">
+                                                    <a href="#">
                                                         <span class="fa fa-search"></span>
                                                         {% trans 'Default tester' %}
                                                     </a>
+                                                    <ul class="dropdown-menu dropdown-menu-right js-toolbar-default-tester" style="min-width: 220px;padding: 5px;">
+                                                        <div style="display: inline-block;">
+                                                            <input type="text" class="form-control typeahead user-field">
+                                                        </div>
+                                                        <button class="btn btn-default" title="Apply" style="margin-left: 5px;" id="default-tester-button">
+                                                            <span class="fa fa-check"></span>
+                                                        </button>
```

Comment:
I think this really needs to be a modal, similar to the existing `window.prompt`. 

Currently it overlaps with other menu entries and will be less versatile for using in other places:
![new_widget_overlap](https://github.com/kiwitcms/Kiwi/assets/1002300/b96a9060-1d4a-451b-b21b-17eb2dbc2a55)